ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -185,48 +186,80 @@ public struct AvatarGroup: View {
+                        let avatarSizeDifference = avatarSize - nextAvatarSize
+                        let sizeDiff = !isLastDisplayed ? (currentAvatarHasRing ? avatarSizeDifference : avatarSizeDifference - ringGapOffset) :
+                            currentAvatarHasRing ? (avatarSize - ringGapOffset) - imageSize : (avatarSize - (ringGapOffset * 2)) - imageSize
+                        let x = avatarSize + tokens.interspace - ringGapOffset
+
+                        let ringPaddingInterspace = nextAvatarHasRing ? interspace - (ringOffset + ringOuterGap) : interspace - ringOffset
+                        let noRingPaddingInterspace = nextAvatarHasRing ? interspace - ringOuterGap : interspace
+                        let rtlRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOuterGap : -x + ringOffset)
+                        let rtlNoRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOffset - ringOuterGap : -x)
+                        let stackPadding = (currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace)
+
+                        let xPosition = currentAvatarHasRing ? x - ringOuterGap - ringOuterGap : x - ringOuterGap
+                        let xPositionRTL = currentAvatarHasRing ? rtlRingPaddingInterspace : rtlNoRingPaddingInterspace
+                        let xOrigin = Locale.current.isRightToLeftLayoutDirection() ? xPositionRTL : xPosition
+                        let yOrigin = sizeDiff / 2
+                        let cutoutSize = isLastDisplayed ? (ringOuterGap * 2) + imageSize : nextAvatarSize
+
+                        VStack {
+                            avatarView
+                                .transition(.identity)
+                                .modifyIf(needsCutout, { view in
+                                    view.mask(Avatar.AvatarCutout(
+                                                xOrigin: xOrigin,
+                                                yOrigin: yOrigin,
+                                                cutoutSize: cutoutSize)
+                                                .fill(style: FillStyle(eoFill: true)))
+                                })
+                        }
+                        .padding(.trailing, tokens.style == .stack ? stackPadding : interspace)
+                        .animation(Animation.linear(duration: 0.1))
+                        .transition(AnyTransition.move(edge: .leading))
+                    }
+                    if overflowCount > 0 {
+                        VStack {
+                            createOverflow(count: overflowCount)
+                        }
+                        .transition(AnyTransition.move(edge: .leading))
+                    }
+                }
+                .frame(height: geo.size.height,
+                       alignment: .leading)
             }
+            .frame(width: calcwidth, height: groupHeight)
+        }
+
+        return avatarGroupContent
+    }
+
+    private var calcwidth: CGFloat {
```

Comment:
The width that is calculated seems wider than needed.
Is that the root cause of the alignment issue?

![Screen Shot 2021-11-19 at 10 41 13 AM](https://user-images.githubusercontent.com/68076145/142674736-f113d253-52f8-4479-bda0-1fce5ddefc6b.png)